packages/mask/src/plugins/Pets/SNSAdaptor/ImgLoader.tsx

```diff
@@ -3,12 +3,16 @@ import { ImgHTMLAttributes, useState } from 'react'
 
 export function ImgLoader(props: ImgHTMLAttributes<HTMLImageElement>) {
     const [loaded, setLoaded] = useState(false)
-
+    const defaultIcon = new URL('../assets/defaultIcon.png', import.meta.url).toString()
+    const onErrorHandle = (e: React.SyntheticEvent<HTMLImageElement, Event>) => {
```

Comment:
```suggestion
    const onErrorHandle = (event: SyntheticEvent<HTMLImageElement, Event>) => {
```

plz do not use `React.*` namespace